sci-libs/primegen/primegen-0.0.97.recipe

```diff
@@ -0,0 +1,61 @@
+SUMMARY="Fast C/C++ prime number generator library"
+DESCRIPTION="primegen is a small, fast library to generate prime numbers \
+in order. It generates the 50847534 primes up to 1000000000 in just 8 \
+seconds on a Pentium II-350; it prints them in decimal in just 35 seconds. \
+primegen can generate primes up to 1000000000000000, although it is not \
+optimized for primes past 32 bits. It uses the Sieve of Atkin instead of the \
+traditional Sieve of Eratosthenes."
+HOMEPAGE="https://cr.yp.to/primegen.html"
+COPYRIGHT="1999, D. J. Bernstein"
+LICENSE="Public Domain"
+REVISION="1"
+SOURCE_URI="https://github.com/ohnx/haiku-primegen/archive/$portVersion.tar.gz"
+SOURCE_DIR="haiku-primegen-$portVersion"
+CHECKSUM_SHA256="9eb6945fbf9f632eaed680b134025e84cb52703c0da1549e1dc14b3a47e6b497"
```

Comment:
@ohnx, please swap SOURCE_DIR and CHECKSUM_SHA256 to meet the recommended order.
See [HaikuPorter-Guidelines#ordering](https://github.com/haikuports/haikuports/wiki/HaikuPorter-Guidelines#ordering)
Thanks!